contrib/TestHarness2/test_harness/summarize.py

```diff
@@ -396,6 +397,10 @@ def done(self):
         if self.was_killed:
             child = SummaryTree('ExternalTimeout')
             child.attributes['Severity'] = '40'
+            if self.long_running:
+                # debugging info for long-running tests
+                child.attributes['LongRunning'] = '1'
+                child.attributes['Runtime'] = str(self.runtime)
```

Comment:
Does it make sense to just always add a Runtime attribute?

Reply:
I think Joshua has other attributes `RealElapsedTime` that people can look at.
This `Runtime` attribute is 0 if it is stopped by _Joshua_ and none-zero if it's passing the `subprocess` timeout,
which seems not very useful for other tests.